extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -330,6 +299,10 @@ export class AmpA4A extends AMP.BaseElement {
      * {?HTMLIframeElement}
      */
     this.iframe = null;
+
+    for (const signingServiceName of this.getSigningServiceNames()) {
+      signatureVerifierFor(this.win).loadKeyset(signingServiceName);
```

Comment:
We call signatureVerifierFor(this.win) multiple times; maybe store it as a local variable instead?